test/e2e/apps/job.go

```diff
@@ -49,9 +49,20 @@ var _ = SIGDescribe("Job", func() {
 		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "failed to ensure job completion in namespace: %s", f.Namespace.Name)
 	})
 
+<<<<<<< HEAD
```

Comment:
Plz fix rebasing codelines.

Reply:
The concern addressed with recent commit